scripts/library.sh

```diff
@@ -235,7 +235,7 @@ function acquire_cluster_admin_role() {
 function report_go_test() {
   # Run tests in verbose mode to capture details.
   # go doesn't like repeating -v, so remove if passed.
-  local args=("${@/-v}")
+  local args=("${@/#-v}")
```

Comment:
Thanks for catching this. However, this will only remove `-v` if it's at the front of the string. I suggest doing:

```
local args=" $@ "
local go_test="go test -race -v ${args/ -v / }"
```

This will correctly remove the first `-v` flag in the arguments.